content/tips-on-applying-for-teacher-training.md

```diff
@@ -0,0 +1,180 @@
+not convinced the admissions team that they’re right for it.
+
+Talk to an experienced teacher for help preparing your application.
+
+## Choose your referees
+
+You need 2 references to apply.
+
+Unlike a standard work reference, your references for teacher training need to
+be named individuals rather than (for example) a human resources team or
+academic department.
+
+Your referees will be asked to write 500 words about your character and
+suitability for teaching.
+
+They’ll have 2 weeks to respond, so it’s worth contacting them before you apply
+so they understand why you’re applying and what they’ll be asked to do.
+
+Choose referees who’ll endorse your:
+
+* teaching passion and potential
+* suitability for working with children
+* academic abilities
+* reliability and professionalism
+
+Ideal referees could include:
+
+* your university tutor or supervisor
+* your current line manager at work
+* your previous employer
+* a teacher at a school where you work or volunteer
+* a supplier or client you’ve worked with (if you’re self-employed)
+
+Referees should not be family members, partners or friends.
+
+Training providers will only accept a character reference as a second reference.
+
+It’s important to have at least one academic or professional reference.
+
+If you’re applying for a salaried course, one of your references must be from
+an employer.
+
+If you’re struggling, get free one-on-one support with choosing your referees.
+
+## Your personal statement
+
+Explaining why you want to be a teacher is the crux of your application - so
+it’s worth taking your time on it.
+
+A personal statement is usually about 500 words. It’s your chance to show your
+motivation, commitment and teaching potential.
+
+* You may want to cover some of the following:
+* why you want to teach
+* what inspired you to choose teaching
+* your thoughts on children and young children’s wellbeing and the education system
+* your understanding of the demands and rewards of the profession
+* personal qualities that will make you a good teacher
+* your contribution to the life of the school outside the classroom – for
+  example, running extra-curricular activities and clubs
+* details any experience you have working with children and what you learnt
+* your suitability to teach a subject or group
+* evidence of the knowledge you have in your subject
+
+Think about <mark>who and what to teach</mark> before you apply.
+
+If you’re invited to interview
+
+Interviews usually last a day and may include time spent teaching real pupils.
+
+You can [get support preparing for an interview from an adviser](/tta-service).
+
+Interviews vary from provider to provider – you’ll be given all the information
+you need when you’re invited to attend.
+
+Some providers are doing interviews online due to COVID-19 restrictions.
+
+## If you get an offer
+
+Congratulations!
+
+Before you start your training provider will check whether you’re safe to work
```

Comment:
there is a typo here, think it should be something like 'Before you start your training, the provider will check...'

Reply:
good spot! feel free to edit @HettieS @peteryates